spec/unit/tables/table_module_spec.rb

```diff
@@ -43,7 +43,7 @@ def custom_cell
 
   def default_cell_height
     return UITableViewAutomaticDimension if TestHelper.ios8
-    return 44.0 if TestHelper.ios7
+    return 97.0 if TestHelper.ios7 # Normally 44, but 97 because of `row_height` designation
```

Comment:
I debated on this one, since it's really test-specific. The TestTableScreen has an estimated row height of 97.0 which gets applied to all cells when you're running in iOS 7 mode. But the true default is 44.0. In the end, I'm low on time, so I just updated this and moved on.